sdk/samples/iot/README.md

```diff
@@ -328,9 +328,9 @@ This section provides an overview of the different samples available to run and
 
 ### IoT Hub Plug and Play Multiple Component Sample
 
```

Comment:
Are we still calling it IoT Hub Plug and Play or IoT Plug and Play?

Reply:
nope good catch